src/main/java/org/broadinstitute/consent/http/db/DarCollectionDAO.java

```diff
@@ -33,6 +33,9 @@
       " AND (" +
         "COALESCE(i.institution_name, '') ~* :filterTerm " +
         " OR (dar.data #>> '{}')::jsonb ->> 'projectTitle' ~* :filterTerm " +
+        //NOTE: since users can change their name, however the dar researcher attribute will not be updated
+        //Search on both dar.data and user displayname for proper name matching
+        " OR (dar.data #>> '{}')::jsonb ->> 'researcher' ~* :filterTerm " +
```

Comment:
We really don't want to be using the `researcher` field in `dar.data` ... it will go away fairly soon. All user-specific data should be pulled from the user's properties. Especially since a user _can_ change their name, their updated name will not be in `dar.data`, but will be on the user.